README.md

```diff
@@ -8,7 +8,7 @@
 
 **Support:**
  - ✅ Public-Verifiable tokens (Blind-RSA)
- - 🚧 Private-Verifiable tokens (VOPRF)
+ - ✅ Private-Verifiable tokens (VOPRF)
```

Comment:
```suggestion
 - ✅ Private-Verifiable tokens (VOPRF)
 - 🚧 Greasing tokens
```

Privacy Pass Protocol draft 16 refers to Privacy Pass Auth scheme draft 14 which defines [grease tokens](https://www.ietf.org/archive/id/draft-ietf-privacypass-auth-scheme-14.html#name-greasing). These could be added to the library.

Reply:
let's do that in another PR